src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+  // additional reducers could be added here
+});
+
+const store = createStore(reducer, applyMiddleware(logger));
+
+export default store;
```

Comment:
- [ ] The redux store  the **combineReducer**  doesn't store any data entry, please fix it